### PR TITLE
Kia: add missing EU EV6 FW versions

### PR DESCRIPTION
**Pull request by razem-io**
Route with OP engaged: ba3e03ccb4b26329|2022-09-08--23-00-28--0

### CODE REVIEW

**Comment by sshane** [comment]
Hey, can you get a route on master or master-ci? The VIN fix isn't on release3 yet

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -1273,6 +1273,7 @@ class Buttons:
   },
   CAR.KIA_EV6: {
     (Ecu.abs, 0x7d1, None): [
+      b'\xf1\x00CV  IEB \x03 101!\x10\x18 58520-CV100',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:1276]:
```suggestion
      b'\xf1\x00CV  IEB \x03 101!\x10\x18 58520-CV100',
      b'\xf1\x8758520CV100\xf1\x00CV  IEB \x03 101!\x10\x18 58520-CV100',
```

**Comment by razem-io** [comment]
> Can you get a route on master or master-ci? The VIN fix isn't on release3 yet

Hey, sorry for the late response. Somehow forgot to respond. I can test this on the weekend. Is it still relevant despite being merged already?

**Comment by sshane** [comment]
It wasn't relevant to this PR, but it would be good to know the Vin query fixed another car!

**Pull request merged**